src/plugins/intel_gpu/include/intel_gpu/primitives/reorder.hpp

```diff
@@ -143,6 +146,8 @@ struct reorder : public primitive_base<reorder> {
     std::vector<float> subtract_per_feature;
     /// @brief Mode of mean execution
     reorder_mean_mode mean_mode;
+    /// @brief Convert truncation Mode
+    bool truncate;
```

Comment:
Shouldnt' it have default value?

Reply:
yes because there are another constructor

Reply:
Updated as your comment, thanks